dandiapi/api/tests/test_stats.py

```diff
@@ -33,10 +36,23 @@ def test_stats_published(api_client, published_version_factory):
 
 @pytest.mark.django_db()
 def test_stats_user(api_client, user):
+    # Reset the User table for test
+    User.objects.all().delete()
+
+    # Create users with different statuses
+    approved_user_count = 0
+    for status in UserMetadata.Status.choices:
+        status_value = status[0]
+        user = User.objects.create(username=f'{status_value.lower()}_user')
+        UserMetadata.objects.create(user=user, status=status_value)
+        if status_value == UserMetadata.Status.APPROVED:
+            approved_user_count += 1
+
     stats = api_client.get('/api/stats/').data
 
-    # django-guardian automatically creates an AnonymousUser
-    assert stats['user_count'] == 2
+    # Assert that the user count only includes users with APPROVED status
+    assert stats['user_count'] == 1
```

Comment:
```suggestion
```

This assertion is no longer useful, nor even correct (in general; see suggestion in the block above that creates users).

Reply:
Updated